README.Rmd

```diff
@@ -208,27 +126,7 @@ The `parse_income_range` function provides some tools for extracting numbers fro
 ```{r}
 
 parse_income_range("$1-$199 ($1-$10,399)", limit = "lower")
```

Comment:
This example should also be removed from the README.

Reply:
Agree. I'll do this when I re-visit the README this week.